go.mod

```diff
@@ -0,0 +1,83 @@
+module apollo.io/apollo-operator
```

Comment:
how about keeping it same with Apollo config domain `www.apolloconfig.com`, e.g. `module apolloconfig.com/apollo-operator`